app/konnect/gateway-manager/declarative-config.md

```diff
@@ -72,6 +74,21 @@ The following steps all use a `.deck.yaml` file to store the
 
 ## Create a configuration file
 
+There are two different methods you can use to obtain the configuration file:
+
+* Copy the autogenerated YAML configuration file from the {{site.konnect_short_name}} UI
+* Create a configuration file with `decK dump`
+
+### Copy the configuration file from the {{site.konnect_short_name}} UI
+
+1. In {{site.konnect_short_name}}, navigate to **Gateway Manager** in the sidebar.
+
+1. Click the control plane you want to copy the configuration file of and click **View configuration**. 
+
+1. Ensure you are viewing the configuration file in YAML format, and then copy and paste the configuration file in a file named `kong.yaml`.
```

Comment:
@a-gabriella Is this correct? I was using Figma for this since I couldn't find it in the UI. Can you copy the config for a whole control plane, or can you only get the config file for entities in a control plane (ex. services, routes, consumers, etc.)?